source/develop/release-management/features/virt/virt-v2v-integration.html.md

```diff
@@ -139,6 +139,33 @@ Import from external environments using virt-v2v in terms of the general flow:
 
 10. Unlock VM & disks
 
+##### Import OVA from VMware
+
+Currently oVirt support importing OVA only from VMware hypervisor.
```

Comment:
yeah.. that's true that it wouldn't work for OVA generated from KVM-based platform (I don't know if it is currently possible to generate one like this though), but if it is possible to generate an OVA from Xen, is there a real reason for it not to work? it could work (with relatively minor adjustments in the worse case) like the OVA from AWS, right?